Formula/manticoresearch.rb

```diff
@@ -0,0 +1,87 @@
+class Manticoresearch < Formula
+  desc "Open source text search engine"
+  homepage "https://www.manticoresearch.com"
+  url "https://github.com/manticoresoftware/manticoresearch/releases/download/3.1.0/manticore-3.1.0-190716-445e806-release.tar.gz"
+  version "3.1.0"
+  sha256 "b8eba31eea5f6f5cf7cb7986c94cb40904f96da99e178c724e7f0081de02388e"
+  head "https://github.com/manticoresoftware/manticoresearch.git"
+
+  depends_on "cmake" => :build
+  depends_on "icu4c" => :build
+  depends_on "libpq" => :build
+  depends_on "mysql@5.7" => :build
+  depends_on "unixodbc" => :build
+  depends_on "openssl"
+  conflicts_with "sphinx",
+   :because => "manticore,sphinx install the same binaries."
+  def datadir
+    var/"manticore/data"
```

Comment:
I'd much rather have this just defined twise

Reply:
Not sure I'm following, is there an issue with `datadir` or `conflicts_with`?

Reply:
I forgot how to type. Anyway, I'd rather get rid of the `def datadir`